src/main/java/com/meilisearch/sdk/json/JacksonJsonHandler.java

```diff
@@ -39,10 +38,7 @@ public String encode(Object o) throws MeilisearchException {
             return (String) o;
         }
         try {
-            // TODO: review later
-            if (o != null && o.getClass() != Key.class) {
```

Comment:
Is this could be considered a problem for other use cases?

Reply:
Not actually I wrote it for this specific case ...

Reply:
Yes, I mean, since the other "regular" use cases didn't need it, they may be affect by it, no?

Reply:
It used it actually but I moved it directly in the Class